src/lenses/RougeScoreLens.tsx

```diff
@@ -0,0 +1,26 @@
+import { Lens } from "../types";
+import rouge from 'rouge';
+
+const calculateRougeScore=(values: unknown[])=>{
+        return rouge.n(values[0], values[1])
+    // return 0;
+}
+
+const RougeScoreLens: Lens = ({values})=>{
+    const result = calculateRougeScore(values)
+    return (<>Rouge score: {result}</>)
```

Comment:
Can you adapt the styling e.G. similar to the [ScalarLense](https://github.com/Renumics/spotlight/blob/main/src/lenses/ScalarLens.tsx#L7)

There is a helper for number formatting at [dataformat.ts](https://github.com/Renumics/spotlight/blob/main/src/dataformat.ts#L140)